README_validations.md

```diff
@@ -0,0 +1,129 @@
+# How to Validate OSCAL
+
+## XML or JSON?
+
+XML files use XML syntax: they are *well-formed* by the rules of XML. (See page on wf vs/and valid.) If you can process your file using a conformant XML processor, it is XML.
+
+Same for JSON (although of course you need a JSON processor, not an XML processor).
+
+But *validating* goes beyond parsing: it tells you not only that a file is can be read successfully, but also that its data structures conform to rules known ahead or stipulated for a particular application of XML or JSON. All XML is well-formed XML, but only OSCAL XML is valid to the OSCAL models. Likewise with JSON and YAML.
+
+If working with XML you will need an XML *schema* (or functional equivalent) to validate your OSCAL. If working with JSON or YAML, you will need a JSON Schema. Normative schemas suitable for validating the various OSCAL models are available on the [project web site]().
+
+## What is my model?
+
+OSCAL comes in several different forms, depending on the data set. At time of writing we have the following models. Each will come with its own schema.
+
+| Model | XML Schema | JSON Schema |
+| -------- | -------- | -------- |
+| Catalog | LINK     | LINK     |
+| Profile | LINK     | LINK     |
+| System security plan (SSP) | LINK     | LINK     |
+| Component definition | LINK     | LINK     |
+| Assessment Plan | LINK     | LINK     |
+| Assessment Results | LINK     | LINK     |
+| Plan of Action & Milestones (POAM) | LINK     | LINK     |
+
+Once you have the model and the schema to validate it with, you are ready.
+
+## OSCAL "all" schema
+
+Additionally, for both XML and JSON we have an "all" schema, which can validate any kind of OSCAL, distinguishing them internally. It is larger and more complex than the model-specific schemas and may not be as efficient or useful; but it can be convenient for certain use cases.
```

Comment:
```suggestion
Additionally, there are "complete" XML and JSON schemas, which can validate any kind of OSCAL. It is larger and more complex than the model-specific schemas and may not be as efficient or useful, but it can be convenient for certain use cases.
```